test/integration/patients/worklist/worklist.js

```diff
@@ -2819,6 +2819,14 @@ context('worklist page', function() {
   specify('action sorting - patient field', function() {
     cy
       .routesForPatientAction()
+      .routeSettings(fx => {
+        const sortingSettings = _.find(fx.data, setting => setting.id === 'sorting');
+        _.each(sortingSettings.attributes.value, function(sortMethod) {
+          sortMethod.sort_type = 'alphanumeric';
+        });
+
+        return fx;
+      })
```

Comment:
I don't know why this happening, but there's a [previous test](https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/integration/patients/worklist/worklist.js#L2375) in this file where `sortMethod.sort_type` is set to `numeric`. 

Which then carries over to this test, where the `sort_type` remains as `numeric` and causes the test to fail intermittently.

This started happening because of the different way we are now importing the fixtures in the `test/support/api/settings.js` file. A change in this PR.

It was originally like this when using `.route()`:

```js
Cypress.Commands.add('routeSettings', (mutator = _.identity) => {
  cy
    .fixture('test/settings').as('fxTestSettings');

  cy.route({
  
    . . .

  })
});
```

Which is changed in this PR to this when using `.intercept()`:

```js
import fxTestSettings from 'fixtures/test/settings';

Cypress.Commands.add('routeSettings', (mutator = _.identity) => {
  cy.intercept({

    . . .

  })
});
```

Reply:
We should ensure we're not introducing mutating elsewhere, rather than add it here I think.  That is an interesting concern though.. we may need to be more aware of mutation where we've been lax up till now in tests.

Reply:
For posterity, instead of doing the above we decided to add a deep clone to the top of `getResource()`:

```js
function getResource(data, type) {
  data = JSON.parse(JSON.stringify(data));

  . . .

}
```